app/uk/gov/hmrc/agentaccesscontrol/controllers/AuthorisationController.scala

```diff
@@ -36,21 +36,20 @@ class AuthorisationController @Inject()(
     val authorisationService: AuthorisationService,
     override val authConnector: AuthConnector,
     val esAuthorisationService: ESAuthorisationService,
-    override val env: Environment,
     cc: ControllerComponents)(implicit val ec: ExecutionContext)
     extends BackendController(cc)
     with AuthAction
     with Logging {
 
   def authorise(authType: String,
-                agentCode: AgentCode,
+                agentCode: String,
```

Comment:
what's the advantage of converting it in code rather than having the AgentCode type in the app.routes file?

Reply:
Given it's only being used in one place it seemed a bit over-engineered so better for readability to just have the conversion in the code.

Reply:
fair enough 👍